### PR TITLE
Add support for IW6x launching

### PR DESCRIPTION
**Pull request by mjkzy**
(before merging, make sure iw6x pull request for X Labs launcher support is merged into dev)

- instead of doing launch-aw and launch-ghosts commands, maybe make a launch-game option to address to S1x, IW6x, and IW4x
- make smaller cards in grid for iw6x?
- maybe put all games in %LOCALAPPDATA%\xlabs\data\games\% instead?

### CODE REVIEW

File: src/launcher/main.cpp [hunk 1]
```diff
@@ -116,6 +116,45 @@ namespace
...
+				{"ghosts-mp", "-multiplayer"},
+			};
+
+			const auto mapped_arg = arg_mapping.find(arg);
+			if (mapped_arg == arg_mapping.end())
+			{
+				return;
+			}
+
+			const auto aw_install = utils::properties::load("ghosts-install");
```
Comment (momo5502) [inline src/launcher/main.cpp:139]: Haha dude. aw_install

File: src/launcher/main.cpp [hunk 2]
```diff
@@ -116,6 +116,45 @@ namespace
...
+			}
+
+			if (!try_lock_termination_barrier())
+			{
+				return;
+			}
+
+			SetEnvironmentVariableA("XLABS_GHOSTS_INSTALL", aw_install->data());
+
+			const auto s1x_exe = get_appdata_path() + "data/iw6x/iw6x.exe";
```
Comment (momo5502) [inline src/launcher/main.cpp:152]: s1x_exe

**Review by momo5502 — APPROVED** [review]
The stuff needs to be changed in another pr, or by me tho :P

**Comment by momo5502** [comment]
Putting all games in one folder is bad as soon as data files come.

**Comment by mjkzy** [comment]
> Putting all games in one folder is bad as soon as data files come.

didnt know data files were coming cant lie

**Pull request merged**